### DISCUSSION TITLE
Initial Beta test reslease

### DISCUSSION THREAD

**Discussion by mdblackledge** [osirixgrpc/osirixgrpc#17] [Announcements]
This is the first beta release of the software. Please let us know of any issues you encounter before August 12th, when the release will be made stable.

<hr /><em>This discussion was created from the release <a href='https://github.com/osirixgrpc/osirixgrpc/releases/tag/v0.2.1-beta0'>Initial Beta test reslease</a>.</em>

**Comment by jgreer0417** [comment]
Hi, I am struggling to get a python test script working, and wanted to ask if I might be missing something obvious.

I have a simple hello world script installed that shows up under the gRPC tasks menu, but I cannot tell if it is running. Is there a terminal that should appear? Nothing appears in the "Task Console" window.

Thanks

**Reply by mdblackledge** [reply]
Hello, and thanks for your interest in OsiriXgrpc!  Would you be happy to share your script? If you have a statement like `print("Hello World!")` run as an installed script, it should print on the task manager.  It would also be helpful if you could raise this as a "Generic" [issue](https://github.com/osirixgrpc/osirixgrpc/issues/new/choose) so that we can dedicate a page towards it's solution   Thanks!
